### DISCUSSION TITLE
how to accelerate the speed of image-encode on windows

### DISCUSSION THREAD

**Discussion by dandingol03** [google/filament#5756] [Q&A] [answered]
HI.
 
i have read the screen pixels using `readPixels` api, and i use `ImageEncoder::encode` to encode pixes into png file. But the encoding cost nearly 1500ms , is there some way to accelerate the spped?

**OS**:
windows

 look forward to you guys reply~

**Comment by bejado** [comment, marked as answer]
Debug builds on Windows are notoriously slow—are you using a release version of Filament?
